src/main/java/frc/robot/RobotContainer.java

```diff
@@ -86,9 +90,16 @@ public RobotContainer() {
   }
 
   private void configureBindings() {
-    shooter.setDefaultCommand(new InstantCommand(() -> shooter.disable(), shooter));
+    // shooter.setDefaultCommand(new InstantCommand(() -> shooter.disable(), shooter));
```

Comment:
This was in place as a safety so that the trigger needs to remain pushed for the shooter to be active.  Was this uncommented just for testing....or needed for the actual control?

Reply:
I commented it for testing.  I don't think the default command should disable/stop motor. The command assigned to trigger can stop/disable the shooter motor.

Reply:
Ok, I am just worried that a command would exit and forget to turn off the shooter...unless there's a reason to do that.  I'm fine with this if you are, though.

Reply:
I think that for now, during bring up type work we should make default commands simple/idle?  We can go back and change default command to stop motor.  I did this because I saw the shooter motor jerking(constant start/stop) and wanted to eliminate any possible places motor is stopped.  Without testing on real hw I don't want to put back the stop motor default.